lifecycle_test.go

```diff
@@ -118,6 +129,18 @@ func createRb(t *testing.T, specName, sourceOption, rbName, rbVersion string, sy
 	assert.NoError(t, lcCli.Exec(argsAndOptions...))
 }
 
+func exportRb(t *testing.T, rbName, rbVersion string) {
+	output := lcCli.RunCliCmdWithOutput(t, "rbe", rbName, rbVersion)
```

Comment:
Might be better to actually check the file was downloaded instead of the response of the command